README.md

```diff
@@ -14,9 +29,21 @@ They caused me some frustration:
 - They could have more helpers (for pagination, rate limit, ...)
 - Typings could make the difference between read/write app
 
-## Goals
```

Comment:
- Number of dubdependencies (from https://npm.anvaka.com/#/view/2d/twitter-lite - 4 for twitter-lite, 7 for twitter-v2, 51 for twit, 50 for twitter) 
- 3-legged authentication

Reply:
Section for sub-dependencies added.
3-legged auth is supported by every other package, and -even its not well documented on each one-, we can't make a comparaison point on this feature :)